ksqldb-rest-client/src/test/java/io/confluent/ksql/rest/client/KsqlClientUtilTest.java

```diff
@@ -184,7 +184,8 @@ public void shouldSerialiseDeserialise() {
     // Then:
     assertThat(buff, is(notNullValue()));
     String expectedJson = "{\"ksql\":\"some ksql\",\"streamsProperties\":{\"auto.offset.reset\":\""
-        + "latest\"},\"requestProperties\":{},\"commandSequenceNumber\":21345,\"sessionVariables\":{}}";
+        + "latest\"},\"requestProperties\":{},\"commandSequenceNumber\":21345,\"sessionVariables\":{},"
+        + "\"maskedKsql\":null}";
```

Comment:
Let's add `@JsonIgnore` to the `maskedKsql` field -- it doesn't need to be part of the serialization.